pkg/transport/grpc/caller.go

```diff
@@ -143,7 +148,7 @@ func (call *Call) SendMsg(ctx context.Context, rw transport.ResponseWriter, pr *
 	ctx, cancel := context.WithCancel(ctx)
 	defer cancel()
 
-	stream, err := grpc.NewClientStream(ctx, proxy, conn, method.fqn)
+	stream, err := grpc.NewClientStream(ctx, proxy, conn, "/"+method.String())
```

Comment:
maybe better to wrap the `"/"+method.String()` implementation inside a function to express the usage? ex: `method.FQN()`.

Reply:
it is only a single place where we add a slash, but sure